main.js

```
@@ -6,7 +6,8 @@ const Handlebars = require('handlebars');
 const INPUT_DATA_TYPES = `{
         to: String,
         subject: String,
-        text: String,
+        text: Maybe String,
```

We should raise an error when text and html not set.